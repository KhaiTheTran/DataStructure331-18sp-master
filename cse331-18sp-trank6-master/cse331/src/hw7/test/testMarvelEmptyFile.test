# Test Dijkstra's algorithm with an empty file

LoadGraph marvelNull marvelNull.tsv

ListNodes marvelNull

FindPath marvelNull node_a node_b