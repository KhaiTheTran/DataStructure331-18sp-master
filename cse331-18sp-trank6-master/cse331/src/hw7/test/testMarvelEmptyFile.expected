# Test Dijkstra's algorithm with an empty file

loaded graph marvelNull

marvelNull contains:

unknown character node a and node b